src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -597,5 +597,14 @@ public static class Diagnostics {
 			defaultSeverity: DiagnosticSeverity.Error,
 			isEnabledByDefault: true
 		);
+
+		public static readonly DiagnosticDescriptor EventMemberMutable = new DiagnosticDescriptor(
+			id: "D2L0070",
+			title: "Event members are not allowed in immtuable types.",
+			messageFormat: "Event members hold registered event handlers. We consider this to be a source of mutability.",
+			category: "Immtubility",
```

Comment:
```suggestion
			category: "Immutability",
```

(nit: maybe a shared `const` would be good for this?)

Reply:
Probably a good idea -- @omsmith and I were talking about having DI pick the analyzers to run based on this category (by looking at their `SupportedDiagnostics`)